production/jaeger-spark-dependencies-job.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v2alpha1
+kind: CronJob
+metadata:
+  name: jaeger-spark-dependencies
+  labels:
+    app: jaeger
+    jaeger-infra: jaeger-spark-dependencies
+spec:
+  schedule: "55 23 * * *"
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          name: jaeger-spark-dependencies
+        spec:
+          containers:
+          - name: jaeger-spark-dependencies
+            image: jaegertracing/spark-dependencies
```

Comment:
For consistency with the other templates, add the version to the image spec.

Reply:
:fearful: There is only `latest`